Frameworks/CoreGraphics/DWriteWrapper.mm

```diff
@@ -195,22 +224,8 @@ CFStringRef _CFStringFromLocalizedString(IDWriteLocalizedStrings* localizedStrin
         CFStringCreateWithCharacters(kCFAllocatorSystemDefault, reinterpret_cast<UniChar*>(wcharString.data()), length));
 }
 
-/**
- * Helper method to retrieve font family names installed in the system.
- *
- * @return Immutable array of font family name strings that are installed in the system.
- */
-CFArrayRef _DWriteCopyFontFamilyNames() {
+CFArrayRef __DWriteCopyFontFamilyNamesFromCollection(const ComPtr<IDWriteFontCollection>& fontCollection) {
     woc::unique_cf<CFMutableArrayRef> fontFamilyNames(CFArrayCreateMutable(kCFAllocatorSystemDefault, 0, &kCFTypeArrayCallBacks));
```

Comment:
Using a `const ComPtr<T>&` confers no benefit over using a `T*`, and disallows the use of this function as a C API facet.